x/crosschain/keeper/cctx_orchestrator_validate_inbound.go

```diff
@@ -48,3 +56,44 @@ func (k Keeper) ValidateInbound(
 
 	return &cctx, nil
 }
+
+// CheckMigration checks if the sender is a TSS address and returns an error if it is.
+// If the sender is an older TSS address, this means that it is a migration transfer, and we do not need to treat this as a deposit.
+func (k Keeper) CheckMigration(ctx sdk.Context, msg *types.MsgVoteInbound) error {
```

Comment:
>checks if the sender is a TSS address and returns an error if it is.
Why? We should precise motivation under this check. Also it seems the function name could be made more explicit

Reply:
69bfc25c6b4e2141f99affa4df32a5883cbaddf5